src/nested_pandas/nestedframe/core.py

```diff
@@ -673,6 +673,71 @@ def drop(
             errors=errors,
         )
 
+    def min(
+        self, 
+        exclude_nest: bool = False,
+        numeric_only: bool = False,
+        **kwargs
+    ):
+        """
+
+        Return the minimum value of each column as a series, including nested columns
+        with prefix to indicate the source column.
+
+        This computes the column-wise minimum (axis=0) across base and nested columns.
+        Row-wise minimum (axis=1) are not supported, as reductions along columns
+        are the primary intended behavior for NestedFrame.
+
+        By default, missing values (NaNs) will be skipped in the computation.
+
+        Parameters
+        ----------
+        exclude_nest : bool, default False
+            If set to True,  will exclude the nested structure and
+            only computes the minimum over the base columns
+        numeric_only : bool, default False
+            Include only float, int, boolean columns.
+        **kwargs
+            See the documentation for :func:`min` for complete details
+            on the keyword arguments accepted by
+            :meth:`~pandas.NestedFrame.min`.
+
+        Returns
+        -------
+        pd.Series 
+
+        Examples
+        --------
+
+        """
+
```

Comment:
up front, I think it would be good to check if there are no nested columns, in which case, just return the super().min(**kwargs). That way we can more reliably have min behave exactly like pandas min for "normal" dataframes.